source/javascripts/app/_utm.js

```diff
@@ -0,0 +1,46 @@
+window.addEventListener("DOMContentLoaded", function(event) {
+  var searchParams = new URLSearchParams(window.location.search);
```

Comment:
This is not supported at all on Internet Explorer. I don't care much about capturing UTM in that case, but we should wrap this in a try/catch to not just break the rest of the page.

Reply:
I experimented with adding a try/catch block and providing a random string for the event name, but neither Firefox or Chrome throw the exception and the catch block never executes. The page load continues ok after this (obviously without adding the cookies). This leads me to believe that maybe adding an event listener that doesn't exist doesn't throw an exception?

Reply:
Added a try/catch block for IE